parcel.py

```diff
@@ -39,16 +39,23 @@
 parcel_file_deleted = parcel_signals.signal('parcel-file-deleted')
 
 
-@parcel_views.route('/')
-def index():
-    return flask.render_template('index.html')
+@parcel_views.route('/', defaults={'delivery': 'lots'})
+def index(delivery):
+    return flask.render_template('index.html', **{'delivery': delivery})
 
 
 def get_filter_arguments():
     return {k: v for k, v in flask.request.args.items()
             if k in METADATA and v}
 
 
+@parcel_views.route('/<string:delivery>')
+def switch_delivery(delivery):
```

Comment:
This view is identical to `index`. You can avoid duplicating code by passing an `endpoint` to the `route` decorator. http://flask.pocoo.org/docs/0.11/api/#flask.Flask.route